packages/kit/src/runtime/app/forms.js

```diff
@@ -161,14 +165,29 @@ export function enhance(form_element, submit = () => {}) {
 		let result;
 
 		try {
+			const is_valid_enctype = enctype?.match(
+				/application\/x-www-form-urlencoded|multipart\/form|text\/plain/
```

Comment:
```suggestion
				/application\/x-www-form-urlencoded|multipart\/form-data|text\/plain/
```
Should it be multipart/form-data instead of multipart/form? based on the enctype from [mdn](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/enctype)